packages/core/botpress/src/database/index.js

```diff
@@ -24,10 +24,17 @@ const initializeCoreDatabase = (knex, botpressPath) => {
   return Promise.mapSeries(tables, fn => fn(knex)).then(() => knex.migrate.latest({ directory }))
 }
 
-const createKnex = async ({ sqlite, postgres, botpressPath }) => {
+const createKnex = async ({ sqlite, postgres, botpressPath, logger }) => {
   const commonConfig = {
     useNullAsDefault: true
   }
+  if (!postgres.enabled && process.version.startsWith('v10.')) {
+    logger &&
+      logger.warn(
+        '[Compatibility Notice] You are trying to use node@10.x but it has compatibility issues with sqlite3 package.' +
```

Comment:
Could you add a link to docs on how to enable Postgres here?

Reply:
@slvnperron , done!